lib/cli/parser.js

```diff
@@ -5,15 +5,27 @@ import { sharedArgs, serverArgs, extensionArgs } from './args';
 import { DRIVER_TYPE, PLUGIN_TYPE } from '../extension-config';
 import { rootDir } from '../utils';
 
+
+function makeDebugParser (parser) {
```

Comment:
is this necessary?

Reply:
Yes, otherwise when we run tests of the parser, the entire mocha process will exit instead of just throw an error. In the past, we could include a `debug` option to parser creation, but that option was removed in argparse 2.0, so now we have to do this (or use a subclass of the parser class which overrides `exit` but we're in JS so we can just do this)